bindings/module.cc

```diff
@@ -217,6 +221,17 @@ namespace pyaotriton {
       .def_property_readonly("strides", &aotriton::TensorView<0>::strides)
       .def_property_readonly("data_ptr", &aotriton::TensorView<0>::data_ptr)
       .def_property_readonly("dtype", &aotriton::TensorView<0>::dtype);
+    m.def("get_name_suffix",
+#if AOTRITON_ENABLE_SUFFIX
+#define xstr(s) str(s)
+#define str(s) #s
+          []() -> std::string { return xstr(AOTRITON_NAME_SUFFIX); }
```

Comment:
@xinyazhang I'm sure there's some nuance with the preprocessing flow here, but can you elaborate why we need two macros `xstr` and `str`?

Reply:
@jithunnair-amd https://gcc.gnu.org/onlinedocs/cpp/Stringizing.html